### PR TITLE
Add New Featured Artist: 7_7 news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
7️⃣

### CODE REVIEW

**Review by peppy — APPROVED** [review]

File: news/2023/2023-04-23-new-featured-artist-7-7.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+![](https://assets.ppy.sh/artists/356/header.jpg)
+
+Before even touching music production, [**7\_7**](https://osu.ppy.sh/users/7491106) was one of us.
+
+Peaking at **global rank #5** in osu!taiko, **7\_7** has encountered and endless slew of insane songs through his drum bashing journey, leading him to produce a highly specific style of music that's perfect for rhythm games: *ultra-dense breakcore tracks with intensity that no other songs dare to match.*
+
+**25** of those songs are ready for mapping from [**7\_7**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/356), including the [osu!original](/wiki/osu!_originals) *7\_7 feat. kasane vavzed - 7D* that debuted in the Grand Finals pool of [TWC 2023](/wiki/Tournaments/TWC/2023) (which happened a few minutes ago, by the way).
+
+If you're struggling to map something by **7\_7** (or any of our other Featured Artists), check out the `#featured-artists` channel in the [osu!dev Discord Server](https://discord.gg/ppy).
```
Comment (MChecaH) [inline news/2023/2023-04-23-new-featured-artist-7-7.md:17]:
```suggestion
If you're struggling to map something by **7\_7** (or any of our other Featured Artists), check out the `#featured-artists` channel in the [osu! Discord Server](https://discord.gg/ppy).
```

`dev` should be removed as to stay consistent with the name change + [wiki article changes.](https://osu.ppy.sh/wiki/en/Community/osu%21_Discord_server).

File: news/2023/2023-04-23-new-featured-artist-7-7.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/356/Songs/7_7%20-%20cafca.mp3">
+</audio>
+
+---
+
+20+ more tracks can be found on [**7\_7**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/356), all pre-timed for instantly gratifying mapping journey. Also, do NOT investigate the timing of *7\_7 feat. kasane vavzed - 7D*. You'll regret it.
+
+That's all for this week! Head to `#featured-artists` in the osu!dev server for some quick mapping advice, and track the [@osugame Twitter](https://twitter.com/osugame) if you want to know who's joining our Featured Artist library next week.
```
Comment (MChecaH) [inline news/2023/2023-04-23-new-featured-artist-7-7.md:57]:
```suggestion
That's all for this week! Head to `#featured-artists` in the osu! server for some quick mapping advice, and track the [@osugame Twitter](https://twitter.com/osugame) if you want to know who's joining our Featured Artist library next week.
```

Same as Line 17.

**Review by MChecaH — CHANGES_REQUESTED** [review]
Keeping consistency with the Discord server's name changes.

File: news/2023/2023-04-23-new-featured-artist-7-7.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Anarchy enters our Featured Artist library through top-tier osu!taiko player **7\_7**'s chaotic sound.
+
+![](https://assets.ppy.sh/artists/356/header.jpg)
+
+Before even touching music production, [**7\_7**](https://osu.ppy.sh/users/7491106) was one of us.
+
+Peaking at **global rank #5** in osu!taiko, **7\_7** has encountered and endless slew of insane songs through his drum bashing journey, leading him to produce a highly specific style of music that's perfect for rhythm games: *ultra-dense breakcore tracks with intensity that no other songs dare to match.*
+
+**25** of those songs are ready for mapping from [**7\_7**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/356), including the [osu!original](/wiki/osu!_originals) *7\_7 feat. kasane vavzed - 7D* that debuted in the Grand Finals pool of [TWC 2023](/wiki/Tournaments/TWC/2023) (which happened a few minutes ago, by the way).
```
Comment (Walavouchey) [inline news/2023/2023-04-23-new-featured-artist-7-7.md:15]:
```suggestion
**25** of those songs are ready for mapping from [**7\_7**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/356), including the [osu! original](/wiki/osu!_originals) *7\_7 feat. kasane vavzed - 7D* that debuted in the Grand Finals pool of [TWC 2023](/wiki/Tournaments/TWC/2023) (which happened a few minutes ago, by the way).
```

File: news/2023/2023-04-23-new-featured-artist-7-7.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/356/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### 7\_7 feat. kasane vavzed - 7D
+
+Play through [this brand new osu!taiko map](https://osu.ppy.sh/beatmapsets/1980743#taiko/4113017) mapped by the artist!
```
Comment (Walavouchey) [inline news/2023/2023-04-23-new-featured-artist-7-7.md:29]:
```suggestion
Play through [this brand-new osu!taiko map](https://osu.ppy.sh/beatmapsets/1980743#taiko/4113017) mapped by the artist!
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2023/2023-04-23-new-featured-artist-7-7.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: 7\_7"
```
Comment (Walavouchey) [inline news/2023/2023-04-23-new-featured-artist-7-7.md:3]: escaping is not supported here

```suggestion
title: "New Featured Artist: 7_7"
```

**Review by Walavouchey — APPROVED** [review]

**Review by pishifat — APPROVED** [review]

**Pull request merged**